extensions/amp-story/0.1/amp-story.css

```diff
@@ -14,5 +14,459 @@
+.i-amphtml-story-page-loading-dot:nth-child(2) {
+  animation-delay: 0.5s;
+}
+
+.i-amphtml-story-page-loading-dot:nth-child(3) {
+  animation-delay: 1.0s;
+}
+
+@keyframes grow {
+  from { transform: scale(1); }
+  to { transform: scale(2); }
+}
+
+.i-amphtml-story-page-loading-text {
+  margin: 38px 0 0;
+  font-size: 14px;
+}
+
+amp-story-page.i-amphtml-story-page-shown > .i-amphtml-story-page-loading-screen {
+  opacity: 0;
+  pointer-events: none;
+}
+
+/** Grid level */
+amp-story-grid-layer {
+  bottom: 0 !important;
+  display: grid !important;
+  left: 0 !important;
+  position: absolute !important;
+  right: 0 !important;
+  top: 0 !important;
+  padding: 68px 32px 32px; /* 44 + 24 */
+  z-index: 2 !important;
+}
+
+amp-story-grid-layer * {
+  box-sizing: border-box !important;
+  margin: 0 !important;
+}
+
+.i-amphtml-story-grid-template-fill > :not(:first-child) {
+  display: none !important;
+}
+
+.i-amphtml-story-grid-template-fill > :first-child {
+  bottom: 0 !important;
+  display: block !important;
+  height: auto !important;
+  left: 0 !important;
+  position: absolute !important;
+  right: 0 !important;
+  top: 0 !important;
+  width: auto !important;
+}
+
+.i-amphtml-story-grid-template-fill > amp-anim img,
+.i-amphtml-story-grid-template-fill > amp-img img,
+.i-amphtml-story-grid-template-fill > amp-video video {
+  object-fit: cover !important;
+}
+
+.i-amphtml-story-grid-template-vertical {
+  align-content: start;
+  grid-auto-flow: row !important;
+  grid-gap: 16px;
+  grid-template-columns: 100% !important;
+  justify-content: stretch;
+  justify-items: start;
+  padding: 32px;
+}
+
+.i-amphtml-story-grid-template-vertical > * {
+  width: 100%;
+}
+
+.i-amphtml-story-grid-template-horizontal {
+  align-content: stretch;
+  align-items: start;
+  grid-auto-flow: column !important;
+  grid-gap: 16px;
+  grid-template-rows: 100% !important;
+  justify-content: start;
+  padding: 32px;
+}
+
+.i-amphtml-story-grid-template-thirds {
+  grid-template-rows: repeat(3, 1fr) !important;
+  grid-template-areas: "upper-third"
+                       "middle-third"
+                       "lower-third" !important;
+  padding: 32px;
+}
+
+/** Bookend */
+.i-amphtml-story-bookend {
+  background: rgba(0, 0, 0, 0.85) !important;
+  color: #fff;
+  bottom: 0 !important;
+  display: none !important;
+  left: 0 !important;
+  position: absolute;
+  right: 0 !important;
+  top: 0 !important;
+  padding: 44px 32px 32px;
+  overflow: auto;
+}
+
+.i-amphtml-story-bookend-active .i-amphtml-story-bookend {
+  display: block !important;
+}
+
+.i-amphtml-story-bookend-heading {
+  text-transform: uppercase;
+  font-size: 10px;
+  padding: 0 0 8px;
+  margin: 48px 0 8px;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.25);
+  color: rgba(255, 255, 255, 0.54);
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  letter-spacing: 0.83px;
+  line-height: 1;
+}
+
+.i-amphtml-story-bookend-article-meta {
+  font-family: 'Roboto', sans-serif;
```

Comment:
Is this ok for a general support?

Reply:
If you mean the high level UX decision of using Roboto: `amp-story` has a pretty opinionated default UI as compared to other components.

If you mean whether the font will actually get loaded, we can consider adding an `<amp-font>` tag for it, internally to the component.

Reply:
I'd rather actually somehow do font-display:optional thing for this. But it's ok.

Reply:
But hows does Roboto gets included on pages? Forced by validator? Or we just fallback?